application/play/README.md

```diff
@@ -0,0 +1,38 @@
+# Play Framework application in Docker
```

Comment:
For consistency we should leave out this title and add a banner image.

Reply:
Fixed